### ISSUE TITLE
Add IP address to "login failed" message in log file "xrdp.log"

### ISSUE THREAD

**Issue by clarsen-007** [neutrinolabs/xrdp#2793]
### xrdp version

0.9.17-2ubuntu2

### Detailed xrdp version, build options
```
[20230911-08:21:01] [INFO ] TLS connection established from 192.168.6.11 port 60604: TLSv1.3 with cipher TLS_AES_256_GCM_SHA384
[20230911-08:21:01] [INFO ] xrdp_caps_process_pointer: client supports new(color) cursor
[20230911-08:21:01] [INFO ] xrdp_process_offscreen_bmpcache: support level 1 cache size 7864320 MB cache entries 2000
[20230911-08:21:01] [WARN ] Client Capability: not enough orders supported by client, client wants off screen bitmap but offscreen bitmaps disabled
[20230911-08:21:01] [INFO ] Loading keymap file /etc/xrdp/km-00000409.ini
[20230911-08:21:01] [WARN ] local keymap file for 0x00000409 found and doesn't match built in keymap, using local keymap file
[20230911-08:21:05] [INFO ] connecting to sesman ip 127.0.0.1 port 3350
[20230911-08:21:05] [INFO ] IPv6 not supported, falling back to IPv4
[20230911-08:21:05] [INFO ] xrdp_wm_log_msg: sesman connect ok
[20230911-08:21:05] [INFO ] sesman connect ok
[20230911-08:21:05] [INFO ] sending login info to session manager, please wait...
[20230911-08:21:05] [INFO ] xrdp_wm_log_msg: login failed for display 0
[20230911-08:21:05] [INFO ] login failed for display 0                                              <<<<<<<<<<<<< Possibly add the IP address of the failed login...   multiple access / security possibilities
```

### Operating system & version

Ubuntu 22.04

### Installation method

dnf / apt / zypper / pkg / etc

### Which backend do you use?

xorgxrdp

### What desktop environment do you use?

KDE

### Environment xrdp running on

VM

### What's your client?

In log file

### Area(s) with issue?

Other

### Steps to reproduce

Log file

### ✔️ Expected Behavior

Additional logging - add IP address

### ❌ Actual Behavior

No IP address is shown

### Anything else?

Awesome project

**Comment by matt335672** [comment]
@clarsen-007 - I think this is addressed in #1976 which was incorporated into v0.9.18. You're just missing that on your current version.

**Comment by clarsen-007** [comment]
Awesome - thank you
